### DISCUSSION TITLE
Display data membership on a page

### DISCUSSION THREAD

**Discussion by f14m07** [liedekef/events-made-easy#423] [Q&A] [answered]
I'd like to display on a page the data for one of the memberships of the logged-in user using the custom fields in the 'answer' table.
So I need to choose the membership, then the fields I want to appear in a template.
[eme_mymemberships] lists all the memberships and I only want one. How select only one membership?

**Comment by liedekef** [comment, marked as answer]
Currently this is not possible. The dev version contains a new shortcode that does what you want:
https://github.com/liedekef/events-made-easy/commit/d21c3c233198fde72fec7db35ae860f01344f4dd

**Comment by f14m07** [comment]
I have tested the dev version with success. Thank you for your fast and efficient work
